lib/jobs/ipmi-catalog.js

```diff
@@ -70,35 +58,28 @@ function ipmiCatalogJobFactory(
         var self = this;
         var cmdResp = [];
 
-        waterline.nodes.findByIdentifier(self.nodeId)
-        .then(function (node) {
-            assert.ok(node, 'No node for ipmi catalog');
+        return waterline.obms.findByNode(self.nodeId, 'ipmi-obm-service', true)
+            .then(function (obmSettings) {
+                assert.object(obmSettings, 'No OBM service available.');
+                assert.ok(!_.isEmpty(obmSettings.config), 'No OBM service config available.');
```

Comment:
whether this means the obm will still be pass when inserting an obm without `config`? I think the obm model should prevent those obm service being added.